pywinauto/base_wrapper.py

```diff
@@ -442,6 +443,70 @@ def get_properties(self):
     GetProperties = get_properties
 
     #-----------------------------------------------------------
+    def draw_outline(
+        self,
+        colour='green',
```

Comment:
Vasily, this are the foo arguments. I am sure this not requires spaces around.
...
https://www.python.org/dev/peps/pep-0008/
"Don't use spaces around the = sign when used to indicate a keyword argument or a default parameter value. "